client/src/locales/en.json

```
@@ -634,6 +634,91 @@
+	"integrationsDiscordInfo": "Connect with Discord and view incidents directly in a channel",
+	"integrationsPrism": "Connect Prism to your favorite service.",
+	"integrationsSlack": "Slack",
+	"integrationsSlackInfo": "Connect with Slack and see incidents in a channel",
+	"integrationsZapier": "Zapier",
+	"integrationsZapierInfo": "Send all incidents to Zapier, and then see them everywhere",
+	"invalidFileFormat": "Unsupported file format!",
+	"invalidFileSize": "File size is too large!",
+	"inviteNoTokenFound": "No invite token found",
+	"loginHere": "Login here",
+	"logsPage": {
+		"description": "This page shows the latest 1000 lines of logs from the Checkmate server",
```

Biome (1.9.4)

[error] The key logsPage was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)